api/models/build.js

```diff
@@ -154,29 +154,27 @@ const afterCreate = async (build) => {
   // create relevant build tasks on build create
   // start some of them
   const { SiteBuildTask, BuildTask, BuildTaskType } = build.sequelize.models;
-  try {
-    await SiteBuildTask.createBuildTasks({ build });
-    const buildTasks = await BuildTask
-      .byStartsWhen(BuildTaskType.StartsWhens.Build)
-      .findAll({ where: { buildId: build.id } });
-    await Promise.all(buildTasks.map(async task => task.enqueue()));
-  } catch (err) {
-    console.error(err);
-  }
+  await SiteBuildTask.createBuildTasks({ build });
+  const buildTasks = await BuildTask
+    .byStartsWhen(BuildTaskType.StartsWhens.Build)
+    .findAll({ where: { buildId: build.id } });
+  await Promise.all(buildTasks.map(async task => task.enqueue()));
 };
 
 const afterUpdate = async (build) => {
   // we start certain build tasks on build completion
   const { BuildTask, BuildTaskType } = build.sequelize.models;
   if (build.state === States.Success) {
-    try {
```

Comment:
Where do the errors get handled on a model hook?

Reply:
I had added these a long time ago just as a safety measure during testing; I think it would probably crash the application since the model hooks occur outside the web request function call stack. Want me to restore and use something other than `console.error`?

Reply:
Can we pull in our event model to capture the error and log it with our logger?